### ISSUE TITLE
[Feature Request] Add to `README.md` the relation of this repository to HandsomeMod/HandsomeMod.

### ISSUE THREAD

**Issue by dreirund** [kirdesde/Handsomemod#4]
Ahoj,

can you state in your [`README.md`](https://github.com/kirdesde/Handsomemod/blob/uf896/README.md), how your repository is related to [/HandsomeMod/HandsomeMod/](https://github.com/HandsomeMod/HandsomeMod/)?

I think you have forked it at some point, but it is not clear if one does not know it, so I suggest you add

* what this repo is based on,
* maybe what is it's purpose and
* how it differs.

I in turn have [forked your repository](https://github.com/dreirund/Handsomemod-uf896/), and have tried to provide that information. However, I had to guess how [your repository](https://github.com/kirdesde/Handsomemod/) relates to [/HandsomeMod/HandsomeMod/](https://github.com/HandsomeMod/HandsomeMod/).

**Comment by kirdesde** [comment]
done, see "about"

**Comment by dreirund** [comment]
> done, see "about"

Well, I cannot confirm.

There it is written:

> *This repository is a mirror of https://github.com/HandsomeMod It is for reference only and is not active for check-ins or for reporting issues*

(Btw., I think you mean https://github.com/HandsomeMod/HandsomeMod/)

I see different content and different branch names and different last commit dates on your repository and on https://github.com/HandsomeMod/HandsomeMod/.

Directory `./tools`: Your repo: 2 years ago. HandsomeMod's repo: 26 days ago. At other places your repo is younger

Also, if I clone both and diff, I get a lot of output (note that the ["HandsomeMod/HandsomeMod" repository](https://github.com/HandsomeMod/HandsomeMod/) does not -- according to `git ls-remote` -- have the branch `uf896` that you use):

`diff -qr kirdesde-Handsomemod HandsomeMod-HandsomeMod`:

```
Files kirdesde-Handsomemod/feeds.conf.default and HandsomeMod-HandsomeMod/feeds.conf.default differ
Files kirdesde-Handsomemod/.git/config and HandsomeMod-HandsomeMod/.git/config differ
Files kirdesde-Handsomemod/.git/HEAD and HandsomeMod-HandsomeMod/.git/HEAD differ
Files kirdesde-Handsomemod/.git/index and HandsomeMod-HandsomeMod/.git/index differ
Files kirdesde-Handsomemod/.git/logs/HEAD and HandsomeMod-HandsomeMod/.git/logs/HEAD differ
Only in HandsomeMod-HandsomeMod/.git/logs/refs/heads: master
Only in kirdesde-Handsomemod/.git/logs/refs/heads: uf896
Files kirdesde-Handsomemod/.git/logs/refs/remotes/origin/HEAD and HandsomeMod-HandsomeMod/.git/logs/refs/remotes/origin/HEAD differ
Only in HandsomeMod-HandsomeMod/.git/objects/pack: pack-b30e9d3b4bf769da20a7c5db9e60941bb11cc83e.idx
Only in HandsomeMod-HandsomeMod/.git/objects/pack: pack-b30e9d3b4bf769da20a7c5db9e60941bb11cc83e.pack
Only in kirdesde-Handsomemod/.git/objects/pack: pack-c0fb053fa7f6e96c948f1559b9389cbf796e568a.idx
Only in kirdesde-Handsomemod/.git/objects/pack: pack-c0fb053fa7f6e96c948f1559b9389cbf796e568a.pack
Files kirdesde-Handsomemod/.git/packed-refs and HandsomeMod-HandsomeMod/.git/packed-refs differ
Only in HandsomeMod-HandsomeMod/.git/refs/heads: master
Only in kirdesde-Handsomemod/.git/refs/heads: uf896
Files kirdesde-Handsomemod/.git/refs/remotes/origin/HEAD and HandsomeMod-HandsomeMod/.git/refs/remotes/origin/HEAD differ
Files kirdesde-Handsomemod/package/base-files/files/lib/functions.sh and HandsomeMod-HandsomeMod/package/base-files/files/lib/functions.sh differ
Files kirdesde-Handsomemod/package/boot/uboot-bcm4908/Makefile and HandsomeMod-HandsomeMod/package/boot/uboot-bcm4908/Makefile differ
Files kirdesde-Handsomemod/package/boot/uboot-bcm4908/patches/200-configs-bcm94908-unset-CONFIG_SPL.patch and HandsomeMod-HandsomeMod/package/boot/uboot-bcm4908/patches/200-configs-bcm94908-unset-CONFIG_SPL.patch differ
Files kirdesde-Handsomemod/package/boot/uboot-bcm4908/patches/201-Assume-TPL-support-for-ATF-when-compiling-U-Boot-wit.patch and HandsomeMod-HandsomeMod/package/boot/uboot-bcm4908/patches/201-Assume-TPL-support-for-ATF-when-compiling-U-Boot-wit.patch differ
Files kirdesde-Handsomemod/package/boot/uboot-envtools/files/ramips and HandsomeMod-HandsomeMod/package/boot/uboot-envtools/files/ramips differ
Files kirdesde-Handsomemod/package/boot/uboot-sunxi/Makefile and HandsomeMod-HandsomeMod/package/boot/uboot-sunxi/Makefile differ
Files kirdesde-Handsomemod/package/firmware/qcom-firmware/Makefile and HandsomeMod-HandsomeMod/package/firmware/qcom-firmware/Makefile differ
Only in HandsomeMod-HandsomeMod/package/kernel/mt76/patches: 101-fix-encap-offload-ethernet-type-check.patch
Only in HandsomeMod-HandsomeMod/package/libs/uclibc++/patches: 006-buildsys-shorten-abi-deb-file-names.patch
Files kirdesde-Handsomemod/package/network/services/hostapd/files/hostapd.sh and HandsomeMod-HandsomeMod/package/network/services/hostapd/files/hostapd.sh differ
Files kirdesde-Handsomemod/package/network/services/hostapd/patches/600-ubus_support.patch and HandsomeMod-HandsomeMod/package/network/services/hostapd/patches/600-ubus_support.patch differ
Files kirdesde-Handsomemod/README.md and HandsomeMod-HandsomeMod/README.md differ
Files kirdesde-Handsomemod/scripts/feeds and HandsomeMod-HandsomeMod/scripts/feeds differ
Only in HandsomeMod-HandsomeMod/target/linux/ath79/dts: qca9533_mikrotik_routerboard-952ui-5ac2nd.dts
Only in HandsomeMod-HandsomeMod/target/linux/ath79/dts: qca9533_mikrotik_routerboard-95x.dtsi
Only in HandsomeMod-HandsomeMod/target/linux/ath79/dts: qca9533_mikrotik_routerboard-map-2nd.dts
Files kirdesde-Handsomemod/target/linux/ath79/image/mikrotik.mk and HandsomeMod-HandsomeMod/target/linux/ath79/image/mikrotik.mk differ
Files kirdesde-Handsomemod/target/linux/ath79/mikrotik/base-files/etc/board.d/01_leds and HandsomeMod-HandsomeMod/target/linux/ath79/mikrotik/base-files/etc/board.d/01_leds differ
Files kirdesde-Handsomemod/target/linux/ath79/mikrotik/base-files/etc/board.d/02_network and HandsomeMod-HandsomeMod/target/linux/ath79/mikrotik/base-files/etc/board.d/02_network differ
Files kirdesde-Handsomemod/target/linux/ath79/mikrotik/base-files/etc/hotplug.d/firmware/10-ath9k-eeprom and HandsomeMod-HandsomeMod/target/linux/ath79/mikrotik/base-files/etc/hotplug.d/firmware/10-ath9k-eeprom differ
Files kirdesde-Handsomemod/target/linux/ath79/mikrotik/base-files/etc/hotplug.d/firmware/11-ath10k-caldata and HandsomeMod-HandsomeMod/target/linux/ath79/mikrotik/base-files/etc/hotplug.d/firmware/11-ath10k-caldata differ
Only in HandsomeMod-HandsomeMod/target/linux/bcm4908/base-files/etc: init.d
Only in HandsomeMod-HandsomeMod/target/linux/bcm4908/base-files/etc: uci-defaults
Only in HandsomeMod-HandsomeMod/target/linux/bcm4908/image: bootfs-bcm4908.its
Only in HandsomeMod-HandsomeMod/target/linux/bcm4908/image: bootfs-bcm4912.its
Only in kirdesde-Handsomemod/target/linux/bcm4908/image: bootfs-generic.its
Only in HandsomeMod-HandsomeMod/target/linux/bcm4908/image: bootfs.itsi
Files kirdesde-Handsomemod/target/linux/bcm4908/image/Makefile and HandsomeMod-HandsomeMod/target/linux/bcm4908/image/Makefile differ
Only in HandsomeMod-HandsomeMod/target/linux/bcm4908/patches-5.4: 037-v5.20-0001-arm64-dts-Add-DTS-files-for-bcmbca-SoC-BCM63158.patch
Only in kirdesde-Handsomemod/target/linux/bcm4908/patches-5.4: 037-v5.20-0001-arm64-dts-broadcom-align-gpio-key-node-names-with-dt.patch
Only in HandsomeMod-HandsomeMod/target/linux/bcm4908/patches-5.4: 037-v5.20-0002-arm64-dts-Add-DTS-files-for-bcmbca-SoC-BCM4912.patch
Only in kirdesde-Handsomemod/target/linux/bcm4908/patches-5.4: 037-v5.20-0002-arm64-dts-broadcom-bcm4908-Fix-timer-node-for-BCM490.patch
Only in HandsomeMod-HandsomeMod/target/linux/bcm4908/patches-5.4: 037-v5.20-0003-ARM64-dts-Add-DTS-files-for-bcmbca-SoC-BCM6858.patch
Only in kirdesde-Handsomemod/target/linux/bcm4908/patches-5.4: 037-v5.20-0003-arm64-dts-broadcom-bcm4908-Fix-cpu-node-for-smp-boot.patch
Only in HandsomeMod-HandsomeMod/target/linux/bcm4908/patches-5.4: 037-v5.20-0004-arm64-dts-Add-DTS-files-for-bcmbca-SoC-BCM63146.patch
Only in HandsomeMod-HandsomeMod/target/linux/bcm4908/patches-5.4: 037-v5.20-0005-arm64-dts-Add-DTS-files-for-bcmbca-SoC-BCM6856.patch
Only in HandsomeMod-HandsomeMod/target/linux/bcm4908/patches-5.4: 037-v5.20-0006-arm64-dts-Add-DTS-files-for-bcmbca-SoC-BCM6813.patch
Only in HandsomeMod-HandsomeMod/target/linux/bcm4908/patches-5.4: 037-v5.20-0007-arm64-dts-broadcom-align-gpio-key-node-names-with-dt.patch
Only in HandsomeMod-HandsomeMod/target/linux/bcm4908/patches-5.4: 037-v5.20-0008-arm64-dts-broadcom-bcm4908-Fix-timer-node-for-BCM490.patch
Only in HandsomeMod-HandsomeMod/target/linux/bcm4908/patches-5.4: 037-v5.20-0009-arm64-dts-broadcom-bcm4908-Fix-cpu-node-for-smp-boot.patch
Only in HandsomeMod-HandsomeMod/target/linux/bcm4908/patches-5.4: 037-v5.20-0010-arm64-dts-Add-base-DTS-file-for-bcmbca-device-Asus-G.patch
Only in kirdesde-Handsomemod/target/linux/generic/backport-5.4: 800-v5.20-0001-dt-bindings-leds-add-Broadcom-s-BCM63138-controller.patch
Only in kirdesde-Handsomemod/target/linux/generic/backport-5.4: 800-v5.20-0002-leds-bcm63138-add-support-for-BCM63138-controller.patch
Only in HandsomeMod-HandsomeMod/target/linux/generic/backport-5.4: 845-v6.0-0001-dt-bindings-leds-add-Broadcom-s-BCM63138-controller.patch
Only in HandsomeMod-HandsomeMod/target/linux/generic/backport-5.4: 845-v6.0-0002-leds-bcm63138-add-support-for-BCM63138-controller.patch
Only in kirdesde-Handsomemod/target/linux/generic/backport-5.4: 846-v5.20-0001-dt-bindings-leds-leds-bcm63138-unify-full-stops-in-d.patch
Only in kirdesde-Handsomemod/target/linux/generic/backport-5.4: 846-v5.20-0002-leds-add-help-info-about-BCM63138-module-name.patch
Only in kirdesde-Handsomemod/target/linux/generic/backport-5.4: 846-v5.20-0003-leds-leds-bcm63138-get-rid-of-LED_OFF.patch
Only in HandsomeMod-HandsomeMod/target/linux/generic/backport-5.4: 846-v6.0-0001-dt-bindings-leds-leds-bcm63138-unify-full-stops-in-d.patch
Only in HandsomeMod-HandsomeMod/target/linux/generic/backport-5.4: 846-v6.0-0002-leds-add-help-info-about-BCM63138-module-name.patch
Only in HandsomeMod-HandsomeMod/target/linux/generic/backport-5.4: 846-v6.0-0003-leds-leds-bcm63138-get-rid-of-LED_OFF.patch
Files kirdesde-Handsomemod/target/linux/ipq40xx/base-files/etc/board.d/02_network and HandsomeMod-HandsomeMod/target/linux/ipq40xx/base-files/etc/board.d/02_network differ
Only in kirdesde-Handsomemod/target/linux/msm89xx: base-files
Only in kirdesde-Handsomemod/target/linux/msm89xx: files
Files kirdesde-Handsomemod/target/linux/msm89xx/image/Makefile and HandsomeMod-HandsomeMod/target/linux/msm89xx/image/Makefile differ
Files kirdesde-Handsomemod/target/linux/msm89xx/image/msm8916.mk and HandsomeMod-HandsomeMod/target/linux/msm89xx/image/msm8916.mk differ
Files kirdesde-Handsomemod/target/linux/msm89xx/Makefile and HandsomeMod-HandsomeMod/target/linux/msm89xx/Makefile differ
Files kirdesde-Handsomemod/target/linux/msm89xx/patches-5.10/0083-arm64-dts-add-support-for-Handsome-Openstick-Family.patch and HandsomeMod-HandsomeMod/target/linux/msm89xx/patches-5.10/0083-arm64-dts-add-support-for-Handsome-Openstick-Family.patch differ
Only in HandsomeMod-HandsomeMod/target/linux/ramips/dts: mt7620a_sitecom_wlr-4100-v1-002.dts
Only in HandsomeMod-HandsomeMod/target/linux/ramips/dts: mt7621_yuncore_ax820.dts
Files kirdesde-Handsomemod/target/linux/ramips/image/mt7620.mk and HandsomeMod-HandsomeMod/target/linux/ramips/image/mt7620.mk differ
Files kirdesde-Handsomemod/target/linux/ramips/image/mt7621.mk and HandsomeMod-HandsomeMod/target/linux/ramips/image/mt7621.mk differ
Files kirdesde-Handsomemod/target/linux/ramips/mt7620/base-files/etc/board.d/02_network and HandsomeMod-HandsomeMod/target/linux/ramips/mt7620/base-files/etc/board.d/02_network differ
Files kirdesde-Handsomemod/target/linux/ramips/mt7621/base-files/etc/board.d/02_network and HandsomeMod-HandsomeMod/target/linux/ramips/mt7621/base-files/etc/board.d/02_network differ
Files kirdesde-Handsomemod/target/linux/ramips/mt7621/base-files/etc/hotplug.d/ieee80211/10_fix_wifi_mac and HandsomeMod-HandsomeMod/target/linux/ramips/mt7621/base-files/etc/hotplug.d/ieee80211/10_fix_wifi_mac differ
Files kirdesde-Handsomemod/target/linux/sunxi/image/cortexa7.mk and HandsomeMod-HandsomeMod/target/linux/sunxi/image/cortexa7.mk differ
Only in HandsomeMod-HandsomeMod/tools/libelf/patches: 100-linux-musl.patch
```

So, can you clarify more what those repositories differ in, since they don _not_ seem to be mirrored?

**Comment by kirdesde** [comment]
I've forked the master branch and created the uf896 and added the support for the uf896 version (the handsomemod repo doesn't contain support for that version) 

And i only pushed the branch uf896 to my remote repo. That's it.

Of course there are differences between both repos as i added a new branch + support for the uf896.

After adding support for the uf896 I didn't do a rebase or merge from the handsomemod repo on my fork. So of course it's possible that it have been changed something in the source repo that it missing in my repo. It's forked and it's not being in sync to the source repo.
